docs/writing-rules/rule-syntax.md

```diff
@@ -154,8 +154,34 @@ Note that `focus-metavariable: $ARG` is not the same as `pattern: $ARG`! Using `
 
 In short, `focus-metavariable: $X` is not a pattern in itself, it does not perform any matching, it only focuses the matching on the code already bound to `$X` by other patterns. Whereas `pattern: $X` matches `$X` against your code (and in this context, `$X` only matches expressions)!
 
+#### Including multiple focus metavariables using set intersection semantics
+
+Include more `focus-metavariable` keys with different metavariables under the `pattern` to match results **only** for the overlapping region of all the focused code:
+
+```yaml
+    patterns:
+      - pattern: foo($X, ..., $Y)
+      - focus-metavariable: $X
+      - focus-metavariable: $Y
+```
+
+The formula above is equivalent to:
+
+```yaml
+    pattern-either:
+    - patterns:
+       - pattern: foo($X, ..., $Y)
+       - focus-metavariable: $X
+    - patterns:
+      - pattern: foo($X, ..., $Y)
+      - focus-metavariable: $Y
+```
+
```

Comment:
This part (from 168-179) should be removed, as it describes the set union instead of the set intersection. It is already correctly added in the set union part, so we can just remove this one

Reply:
ok!